Dockerfile.ubuntu

```diff
@@ -0,0 +1,7 @@
+FROM python:3.10.6-slim
+WORKDIR /app
+RUN apt update
+RUN apt install -y wget git libgl1 libglib2.0-0
+COPY ./ ./
+RUN ./webui.sh -f --skip-torch-cuda-test --exit
+CMD ["./webui.sh", "-f", "--listen"]
```

Comment:
Using Docker means that it is stable, controllable and reproducible, but using the installation script directly for initialization will bring countless uncertainties, including dependency versions and so on.